stonesoup/models/transition/nonlinear.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please remove until ready, and raise separate pull request.

Reply:
ok. Whole file removed.